notebooks/vertex_genai/solutions/retrieval_augmented_generation.ipynb

```diff
@@ -0,0 +1,1034 @@
+{
```

Comment:
<p>It's a bit unclear, what would become the "context" and what would become the "query". Maybe provide a bit more detail.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/413/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Done</p>